Examples/Echo.Bus.Tls/Echo.Bus.Tls.csproj

```diff
@@ -51,15 +51,19 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
   </ItemGroup>
   <ItemGroup>
+    <None Include="test-keys\cert.pem" />
+    <None Include="test-keys\privatekey.pem" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\NsqSharp.WindowService\NsqSharp.WindowService.csproj">
+      <Project>{D030B7A2-7E83-4190-81DE-394D04283ADD}</Project>
+      <Name>NsqSharp.WindowService</Name>
+    </ProjectReference>
     <ProjectReference Include="..\..\NsqSharp\NsqSharp.csproj">
-      <Project>{7327244d-e163-4c35-9c1a-8dbdf06b9408}</Project>
+      <Project>{cecbcab8-292d-4833-b580-0353085ea6f7}</Project>
       <Name>NsqSharp</Name>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup>
-    <None Include="test-keys\cert.pem" />
```

Comment:
???

Reply:
`<None Include="test-keys\cert.pem" />` was moved up in the `.csproj` file. This is a TLS example project.